hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/ContainerStateMachine.java

```diff
@@ -862,4 +864,48 @@ public void notifyLeaderChanged(RaftGroupMemberId groupMemberId,
                                   RaftPeerId raftPeerId) {
     ratisServer.handleLeaderChangedNotification(groupMemberId, raftPeerId);
   }
+
+  @Override
+  public String toStateMachineLogEntryString(StateMachineLogEntryProto proto) {
+    try {
+      ContainerCommandRequestProto requestProto =
+              getContainerCommandRequestProto(proto.getLogData());
+      return getWriteChunkInfo(requestProto);
+    } catch (Throwable t) {
+      return "";
+    }
+  }
+
+  private String getWriteChunkInfo(ContainerCommandRequestProto requestProto) {
+    StringBuilder builder = new StringBuilder();
+    Preconditions.checkArgument(requestProto.getCmdType() == Type.WriteChunk);
+
+    long contId = requestProto.getContainerID();
+    WriteChunkRequestProto wc = requestProto.getWriteChunk();
+
+    builder.append("cmd=");
+    builder.append(requestProto.getCmdType().toString());
+
+    builder.append(", container id=");
+    builder.append(contId);
+
+    builder.append(", blockid=");
+    builder.append(contId);
```

Comment:
We can omit this as we are already printing the containerID earlier.

Reply:
done

Reply:
Changed it to read the container ID from the blockID, in case it is different. :)